compiler/src/builder.rs

```diff
@@ -22,7 +22,7 @@ pub struct Builder<'a> {
 }
 
 type LoopDepth = usize;
-pub type TapeCell = i32;
+pub type TapeCell = (i32, i32);
```

Comment:
I feel like we should have an enum here, with a variant for 2D grid cell, but I don't think it's worth changing now.